src/main/java/com/google/alpollo/model/SongInfo.java

```diff
@@ -0,0 +1,38 @@
+package com.google.alpollo.model;
+
+import java.util.List;
+import com.google.alpollo.Song;
+import com.google.alpollo.model.SongEntity;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Entity;
+
+/**
+ * Class used for storing data related to the Natural Language API analysis.
+ * Extending the Song class, it provides all the info related to a specific song.
+ * 
+ * We extend from Song because we do not always need the entire song data. Sasha mostly
+ * needs to work with the "superficial" data (artist name, song name etc.) while Budi has to 
+ * work with the analysis results.
+ */
+@Entity
+public class SongInfo extends Song {
```

Comment:
Similarly to what I wrote in PR #26, this class inheritance does not make sense. With Object Oriented Programming, you need to think of a subclass as being a particular kind of its parent class. But SongInfo is not a Song. If you want to add data to a Song, then add fields inside the Song class.
If you want to keep all those together because they form so sort of logical block, then keep SongInfo (maybe with a more illuminating name), but let it just contain the fields, and not be a subclass of Song. Then Song can have a SongInfo field.
And if you want to still be able to have Song objects that don't have SongInfo, then you can have a constructor that sets this field to null, or set the value of this field through a setter instead of inside the constructor